### PR TITLE
[release-1.17] Retrigger 1.17.0-rc.0 build

### PR DESCRIPTION
**Pull request by johnma14**

### CODE REVIEW

**Comment by ericvn** [comment]
Note that since you didn't publish a rc.0 you can just trigger another build of it. Add a line to the trigger file for example so the repo sees the file change but the value in the file is still the same.

File: release/trigger-build [hunk 1]
```diff
@@ -1 +1 @@
-1.17.0-rc.0
+1.17.0-rc.1
```
Comment (ericvn) [inline release/trigger-build:1]:
```suggestion
1.17.0-rc.0

```

**Comment by johnma14** [comment]
Since we didn't publish rc.0, changing it back to rc.0. Thanks Eric.

**Comment by johnma14** [comment]
@pmerrison ready for you to approve. Thanks.

**Comment by johnma14** [comment]
Closing this in favor of https://github.com/istio/release-builder/pull/1377

**Pull request closed without merging**